roles/contiv_network/tasks/main.yml

```diff
@@ -82,8 +85,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+- name: reload system config for netmaster
+  shell: systemctl daemon-reload
+  when: run_as == "master"
```

Comment:
Why do the reload before setting the state to started/enabled? If the service is not loaded, apparently setting it to enabled fails.